docs/options.md

```diff
@@ -36,7 +65,7 @@ Default value of `prefix` is `/`.
 
 Defines the base URL which is used and prepended to make requests server side requests.
 
-Environment variable `API_URL` can be used to **override** `baseURL`.
+[Environment variable](#environment-variables) `API_URL` can be used to **override** `baseURL`.
```

Comment:
Also here 🙈

Reply:
I noticed right after commit the first one 😂